### PR TITLE
First-step

### PR DESCRIPTION
**Pull request by Ta0o**
45 line
i write command to change empty blank query but it doesn`t work.Also i add some commands but they work good.And i have a question about object destroying.If i understand well,i need to delete all links to this object but i don`t understand how to do it

### CODE REVIEW

**Comment by kosstbarz** [comment]
1) I suggest you name the Pull Request as your branch "First-step".
2) For now you shouldn't worry about deletion of objects. It will be deleted when you don't need it.
3) I didn't understand, what empty query are you about.

**Comment by Ta0o** [comment]
I meen that 
If i dont write anything i write some text,but after simple enter,programm ends.

**Comment by kosstbarz** [comment]
As I can see, it is expected behaviour.

**Comment by Ta0o** [comment]
I thing.i try to change all code into switch construction,expected behavour became expected))

**Comment by Ta0o** [comment]
I checked that switch construction is better to use, but i have some questions about methods.
i need to request a command permanently,but i dont know how to check it permamently.

File: src/core/GameClass.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        Scanner input = new Scanner(System.in);
+        System.out.println("hello!");
+        System.out.println("Welcome to my first game.");
+        System.out.println("This is my first experience in game developing.");
+        System.out.println("There are some commands, which help you to play.");
+        System.out.println("Write (/rules) to see all the commands.");
+
+        command = input.nextLine();
+        System.out.println(command);
+        switch(command) {
```
Comment (Ta0o) [inline src/core/GameClass.java]: Here.I decide to make switch block cause its better to check command here.After this i change default block for 1 method to do it less.

**Comment by kosstbarz** [comment]
And the main problem: Game finishes before I create a game. I want game to finish only after command "quit game".

**Comment by Ta0o** [comment]
I know)Just you say that application must read a command)No others clauses.I understand that this is a common sense, but all the same if we do as you want, I need help in the form of a couple of inundations.

**Comment by Ta0o** [comment]
#8

**Comment by Ta0o** [comment]
#7

**Pull request merged**